dotcom-rendering/src/components/FrontSectionTitle.tsx

```diff
@@ -58,20 +58,25 @@ export const FrontSectionTitle = ({ title, collectionBranding }: Props) => {
 		case 'foundation': {
 			const {
 				branding: { logo },
+				isFrontBranding,
```

Comment:
I don't think we should rely on this flag. This is used only when the container displays the branding on behalf of the entire front. See comment [here](https://github.com/guardian/dotcom-rendering/blob/main/dotcom-rendering/src/types/branding.ts#L49-L52)